dali/operators/reader/loader/video/frames_decoder.cc

```diff
@@ -207,18 +226,18 @@ void FramesDecoder::BuildIndex() {
     entry.is_flush_frame = false;
 
     if (entry.is_keyframe) {
-      last_keyframe = index_.size();
+      last_keyframe = index_.value().size();
```

Comment:
I think you can write `index_->size()` (same with other method calls on `index_`)

Reply:
As far as I understand the documentation, `.value()` performs a check if the value is present, `*` and `->` don't do that so they're tiny bit more efficient in cases you're sure the value is there.

Reply:
Done